Update/omot_012.txt

```diff
@@ -2863,7 +2863,7 @@ void main()
 //r魅音はそう言って、ひらひらと笑顔で手を振る。とりあえず俺はまあいいか、とばかりに小首を傾けてから、扉を開いて廊下に出た。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "魅音はそう言って、ひらひらと笑顔で手を振る。とりあえず俺はまあいいか、とばかりに小首を傾けてから、扉を開いて廊下に出た。",
-			NULL, "Mion waved her arms as she said that and smiled. For the time being, I decided to leave it at that. I just tilted my head, then opened the door and went out into the hallway.", Line_Normal);
+			NULL, "Mion waved her arms as she said that and smiled. For the time being, I just tilted my head instead of saying Whatever, then opened the door and went out into the hallway.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Mion waved her arms as she said that and smiled. For the time being, I just tilted my head instead of saying anything, then opened the door and went out into the hallway.", Line_Normal);
```